frontend/src/app/main/ui/workspace/sidebar/options/rows/color_row.cljs

```diff
@@ -42,15 +41,108 @@
              (* 100)
              (fmt/format-number)))))
 
-(defn remove-multiple
-  [v]
-  (if (= v :multiple) nil v))
+(mf/defc color-info-wrapper*
+  {::mf/private true}
+  [{:keys [color class handle-click-color children select-on-focus opacity on-focus on-blur on-opacity-change]}]
+  [:div {:class (stl/css :color-info)}
+   [:div {:class class}
+    [:div {:class (stl/css :color-bullet-wrapper)}
+     [:> swatch* {:background color
+                  :on-click handle-click-color
+                  :size "small"}]]
+    children]
+   (when opacity
+     [:div {:class (stl/css :opacity-element-wrapper)}
+      [:span {:class (stl/css :icon-text)} "%"]
+      [:> numeric-input* {:value (-> color :opacity opacity->string)
+                          :class (stl/css :opacity-input)
+                          :placeholder "--"
+                          :select-on-focus select-on-focus
+                          :on-focus on-focus
+                          :on-blur on-blur
+                          :on-change on-opacity-change
+                          :data-testid "opacity-input"
+                          :default 100
+                          :min 0
+                          :max 100}]])])
+
+(mf/defc color-token-row*
+  {::mf/private true}
+  [{:keys [active-tokens color-token color on-swatch-click-token detach-token open-modal-from-token]}]
+  (let [active-tokens (if (delay? active-tokens)
```

Comment:
It is not easy to understand what is this function doing and why. Seems a bit confusing because both variables have the same name. Could you add a short comment?